lib/LaTeXML.pm

```diff
@@ -201,6 +201,8 @@ sub convert {
     $$opts{sitedirectory} = $sandbox_directory;
 
     if ($$opts{format} eq 'epub') {
+      # give main document a predictable name, avoid clash with nav.xhtml
+      $sandbox_destination = 'index.xhtml';
```

Comment:
that's ... a bit too standard? if the desired effect is to avoid a clash, why not check for a clash and avoid it? 

I'm unsure what to think about every single ePub generated by latexml having the same name for its main xhtml file.

Reply:
The split file names are also standardised, `Ch1.xhtml`, `Ch1.S1.xhtml` etc, I don't think it's too bad if the main page is always the same.

To be honest, I also did it because generating a custom TOC is virtually impossible because XSLT does not know the name of the file it's creating. It knows the others via `@href`, but not itself. I meant to bring this up at some point.

Reply:
Great piece of information to add here, thanks!

Reply:
The names are standard only if you use id-based naming (and haven't customized the id generation). Someone might use label-based names, for example (although that's more useful on a website than hidden within an ePub)